README.md

```diff
@@ -173,6 +173,10 @@ Default to `true`.
 Flag to enable the Mojave hardened runtime when signing the app.  Disabled by default, requires Xcode >= 10 and
 macOS >= 10.13.6.
 
+`restrictMode` or `restrict` - *Boolean*
```

Comment:
Can this just be `restrict` for simplicity, no need for two names here

Reply:
Can we link to the restrict mode docs here so folks know what it is?

Reply:
Thanks @MarshallOfSound for the review!
would this be a good doc to answer that? https://stackoverflow.com/questions/29665371/compiling-a-binary-immune-to-library-redirection-on-mac-os-x
I also found this https://web.archive.org/web/20161007013145/http://pewpewthespells.com/blog/blocking_code_injection_on_ios_and_os_x.html but the source seems to be 404 now, only readable from web archive. it does have a pdf version that is still available: https://pewpewthespells.com/blog/blocking_code_injection_on_ios_and_os_x.pdf

Reply:
I'd link to https://developer.apple.com/documentation/security/seccodesignatureflags/kseccodesignaturerestrict?language=objc